include/f_core/math/n_math_utils.h

```diff
@@ -0,0 +1,18 @@
+#ifndef N_MATH_UTILS_H
+#define N_MATH_UTILS_H
+
+#include <zsl/orientation/quaternions.h>
+
+namespace NMathUtils {
+    /**
+     * @brief Calculate roll, pitch, and yaw from a quaternion.
+     *
+     * @param[in] inputQuat The input quaternion.
+     * @param[out] roll Output roll angle in radians.
```

Comment:
either this comment is wrong or the implementation is wrong? You have a multiplier of 180/pi which turns radians into degrees in the implementation but say it comes out as radians?